src/backend/marsha/core/api/video.py

```diff
@@ -423,9 +423,7 @@ def end_live(self, request, pk=None):
         else:
             video.delete()
             video.upload_state = defaults.DELETED
-            video.live_state = None
-            video.live_info = None
-            video.live_type = None
+            video.live_state = defaults.STOPPED
```

Comment:
Shouldn't we keep the other live related attributes reset ?
```
video.live_info = None
video.live_type = None
```
Also I don't get why we need to set the live_state as stopped here.

Reply:
Me neither. IMO you should not touch this. The `live_state` is not stopped anymore. You have to handle this case in the front application like we did before IMO